yaml/testlist.yaml

```diff
@@ -205,3 +207,11 @@
     +enable_irq_seq=1
   compare_opts: >
     +compare_final_value_only=1
+
+- test: riscv_csr_test
+  description: >
+    Test all CSR instructions on all implemented CSR registers
+  iterations: 5
```

Comment:
Can you change it to 1 iteration by default?
BTW, we should update the README.md as the CSR script requires additional package to run.